packages/kolibri-common/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+import AppError from '../../kolibri/core/assets/src/views/AppError';
```

Comment:
My thought was that we could just put the files themselves in the package, wouldn't even need the index.js then, as could just import `kolibri-common/components/GlobalSnackbar` or somesuch.

Reply:
Ahh i see. Let me refactor this. Thanks for this clarification

Reply:
Ahh i see. Let me refactor this. Thanks for this clarification